content/09.evolution.md

```diff
@@ -138,9 +97,95 @@ While it has been suggested that more thorough investigation of the origins of C
 A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of more thorough monitoring of zoonotic diseases.
 More importantly, it would allow researchers to have a better understanding of and to community concerns about potential emerging viral threats.
 
-### Question 3: Who is Affected?
-
-#### Genetic Contributors to Susceptibility
+### Question 3: Which Species Are Susceptible?
+
+Given the strong evidence for a zoonotic origin of SARS-CoV-2, another evolutionary question that received significant attention, especially early on, was whether humans could infect other species with SARS-CoV-2.
+In the modern age, opportunities for human-to-animal transmission events could arise in interactions with companion animals, zoo animals, house pests, and livestock.
```

Comment:
```suggestion
In the modern age, opportunities for human-to-animal transmission events could arise in interactions with companion animals, zoo animals, house pests, hunting, urbanized wildlife, and livestock.
```

Reply:
OMG how could I forget urban wildlife, that was a huge part of my PhD :laughing: